src/test/kotlin/nl/avisi/structurizr/site/generatr/site/model/SoftwareSystemPageViewModelTest.kt

```diff
@@ -85,7 +85,7 @@ class SoftwareSystemPageViewModelTest : ViewModelTest() {
 
     @TestFactory
     fun `active tab`() = Tab.entries
-        .filter { it != Tab.COMPONENT } // Component link is dynamic
+        .filter { it != Tab.COMPONENT && it != Tab.CODE } // Component & code links are dynamic
```

Comment:
Looks like we are also missing a `code views tab only visible when code image views available` test. Would you mind adding that?